sw/host/tests/chip/rv_dm/src/control_status.rs

```diff
@@ -49,7 +49,10 @@ fn test_control_status(opts: &Opts, transport: &TransportWrapper) -> Result<()>
     dmi.dmi_write(consts::DMCONTROL, dmcontrol)?;
     // Since this target only supports 1 hart, the WARL behavior of this register is such that no
     // bits should be set at this point, except for dmactive.
-    assert_eq!(dmi.dmi_read(consts::DMCONTROL)?, consts::DMCONTROL_DMACTIVE_MASK);
+    assert_eq!(
```

Comment:
If you want to correct the formatting, maybe make that part of a different commit in the PR?

Reply:
reverted